src/Codegen/Constraints/StringBuilder.php

```diff
@@ -146,6 +168,9 @@ protected function getCheckMethod(): CodegenMethod {
   <<__Override>>
   public function getType(): string {
     if (Shapes::keyExists($this->typed_schema, 'hackEnum')) {
+      if ($this->typed_schema['generateHackEnum'] ?? false) {
+        return $this->typed_schema['hackEnum'];
```

Comment:
Hmm it’s a bummer that we have to fork here. I bet we could update the pre-existing branch to drop the leading `\` without causing issues, though it would require regenerating a lot of code.